contracts/axelarnet-gateway/src/contract/execute.rs

```diff
@@ -74,30 +76,28 @@ impl CallContractData {
     }
 }
 
+enum RoutingDestination {
+    Nexus,
+    Router,
+}
+
+type Result<T> = error_stack::Result<T, Error>;
+
 pub fn call_contract(
```

Comment:
Update the doc in `msg.rs` for CallContract to document behaviour differences depending on chain type / attached token